modin/pandas/base.py

```diff
@@ -3127,6 +3127,56 @@ def __init__(
         ]
         self.__groups = self.__get_groups(*self.resample_args)
 
+    _attributes = [
```

Comment:
Propose to not create additional list. You could modify `self.resample_args` above to make it dict. In that case you also will fix issue describet in FIXME section above.